api/swagger.json

```diff
@@ -0,0 +1,1304 @@
+    "version": "0.1.0",
+    "title": "Chapter",
+    "termsOfService": "https://www.freecodecamp.org/news/terms-of-service/",
+    "contact": {
+      "email": "chapter+dev@freecodecamp.org"
+    },
+    "license": {
+      "name": "BSD 3-Clause",
+      "url": "https://github.com/freeCodeCamp/chapter/blob/master/LICENSE"
+    }
+  },
+  "tags": [
+    {
+      "name": "chapter",
+      "description": "Everything about your Chapters",
+      "externalDocs": {
+        "description": "Find out more",
+        "url": "http://chapter.io"
+      }
+    },
+    {
+      "name": "location",
+      "description": "Everything about your Locations",
+      "externalDocs": {
+        "description": "Find out more",
+        "url": "http://chapter.io"
+      }
+    },
+    {
+      "name": "user",
+      "description": "Everything about your Users",
+      "externalDocs": {
+        "description": "Find out more",
+        "url": "http://chapter.io"
+      }
+    }
+  ],
+  "paths": {
+    "/locations": {
+      "post": {
+        "tags": [
+          "location"
+        ],
+        "summary": "Add a new location to the map",
+        "description": "",
+        "operationId": "addLocation",
+        "requestBody": {
+          "content": {
+            "application/json": {
+              "schema": {
+                "$ref": "#/components/schemas/Location"
+              }
+            }
+          },
+          "description": "Location coordinates",
+          "required": true
+        },
+        "responses": {
+          "200": {
+            "description": "Location response",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/Location"
+                }
+              }
+            }
+          },
+          "401": {
+            "description": "Not authorized"
+          },
+          "400": {
+            "description": "Invalid Location object"
+          },
+          "default": {
+            "description": "unexpected error",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/Error"
+                }
+              }
+            }
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "location"
+        ],
+        "summary": "Finds Locations by city, country etc",
+        "description": "Multiple status values can be provided as query params",
+        "operationId": "findLocations",
+        "parameters": [
+          {
+            "name": "city",
+            "in": "query",
+            "description": "City value that needs to be considered for filter",
+            "required": false,
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "name": "country",
+            "in": "query",
+            "description": "Country value that needs to be considered for filter",
```

Comment:
Is this a two-letter (e.g. iso 3166-1 alpha-2) code, long-form name, or something else? Thinking from a client/consistency perspective going with an alpha-2 code is more predictable across the board.